lib/gscraper_web/templates/registration/new.html.eex

```diff
@@ -0,0 +1,27 @@
+<div class="card card-auth">
+  <div class="card-auth__title">
+    <h4><%= dgettext "auth", "Sign up" %></h4>
+  </div>
+  <form class="form form-auth">
+    <div class="form__group required">
+      <label for="username" class="required">Username</label>
+      <input class="form__control" required="required" autofocus="autofocus" type="text" name="email">
+      <div class="form__feedback"></div>
+    </div>
+    <div class="form__group required">
+      <label for="password" class="required">Password</label>
+      <input class="form__control" required="required" autofocus="autofocus" type="password" name="password">
```

Comment:
Should we use `autofocus` only in one input? maybe `username`

Reply:
yes, definitely. Removed in 970fc61.